bundles/sirix-core/src/main/java/org/sirix/service/xml/shredder/WikipediaImport.java

```diff
@@ -572,7 +574,7 @@ public static void main(final String[] args) throws SirixException {
         eventFactory.createStartElement(new QName(NSP_URI, "text", XMLConstants.DEFAULT_NS_PREFIX), null, null);
 
     // Create list.
-    final List<StartElement> list = new LinkedList<StartElement>();
+    final List<StartElement> list = new LinkedList<>();
```

Comment:
While you're at it: can't this be changed to an `ArrayList<>(5)` or an `ImmutableList.of(...)`, so that we get a perfectly sized array, rather than a scattered node-backed list (which is less read-performant)?

Reply:
Now you can simply use `List.of(...)`.

Reply:
Thanks @JohannesLichtenberger I guess I'm becoming outdated in the Java world 😅